main.go

```diff
@@ -20,6 +22,31 @@ func main() {
 		return
 	}
 
+	latestVersion, err := utils.GetLatestVersion()
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	if utils.Version != latestVersion {
```

Comment:
I wonder if there's a way we could only ask this once a day or something like that?

Reply:
In my opinion that might defeat the purpose. Its easy enough to upgrade. Its probably safe as well to have everyone on the updated Binaries in case procedures or commands have changed.

Reply:
+1 to Raf's comment - I'm in favour of being annoying to ensure we're not missing functionality